config/application.rb

```diff
@@ -14,17 +14,6 @@
 
 module Upaya
   class Application < Rails::Application
-    config.generators do |g|
-      g.test_framework :rspec,
-                       fixtures: true,
-                       view_specs: false,
-                       helper_specs: false,
-                       routing_specs: false,
-                       controller_specs: false,
-                       request_specs: false
-      g.fixture_replacement :factory_girl, dir: 'spec/factories'
-    end
```

Comment:
ah you're right. my squash of 30+ commits must have picked up an extra one. Does master look ok, since this PR was already merged?